### DISCUSSION TITLE
Missing HQQ/LoRA example -- can you please add to repo?

### DISCUSSION THREAD

**Discussion by ColumbusAI** [mobiusml/hqq#117] [General]
Appreciate the great work and I'd like to quantize a model using HQQ+. The Readme mentions an example but it is not in the repo:

"_We provide a complete example to train a model with HQQ/LoRA that you can find in examples/lora/train_hqq_lora_example.py._"

**Comment by mobicham** [comment]
Hi ! Yeah you are right, I removed  it just a few days ago because it was an old deprecated example. I just added a new one: https://github.com/mobiusml/hqq/blob/master/examples/lora/hqq_plus.py 
I can't reproduce exactly the same results as I got with my custom code (I don't use HF peft), so it would require some additional knob tweaking in terms of learning rate/optimizer scheduling etc.

Also, see this conversation which addresses some of the issues, in case you encounter similar problems: https://github.com/mobiusml/hqq/issues/115

Hope it helps!

**Reply by ColumbusAI** [reply]
Thanks @mobicham
